crates/ruff_python_formatter/tests/snapshots/black_compatibility@py_310__pattern_matching_extras.py.snap

```diff
@@ -248,19 +204,16 @@ match bar1:
          pass
  
  
- match bar1:
--    case Foo(aa=Callable() as aa, bb=int()):
-+    case NOT_YET_IMPLEMENTED_PatternMatchClass(0, 0):
-         print(bar1.aa, bar1.bb)
-     case _:
-         print("no match", "\n")
- 
+@@ -114,6 +125,9 @@
  
  match bar1:
--    case Foo(
+     case Foo(
 -        normal=x, perhaps=[list, {"x": d, "y": 1.0}] as y, otherwise=something, q=t as u
--    ):
-+    case NOT_YET_IMPLEMENTED_PatternMatchClass(0, 0):
++        normal=x,
++        perhaps=[list, {"NOT_YET_IMPLEMENTED_PatternMatchMapping": _, 2: _}] as y,
```

Comment:
Okay, the problem here is that `PatternMatchMapping` isn't yet implemented and the text emitted by the "fake" implementation is much longer than the expected string.

We can ignore this.